Clarify architecture overview (#4144)
Particular/docs.particular.net: platform/serviceinsight.include.md

> ServiceInsight is a desktop application with features tailored to developers. It allows for advanced debugging, tracking the flow of an individual message in the system, observing sagas, and more.

Reviewer: ```suggestion
ServiceInsight is a desktop application designed for developers. It provides advanced debugging, tracking the flow of an individual message in the system, observing sagas, and more.
```
